.github/workflows/pythonapp.yml

```diff
@@ -41,15 +41,15 @@ jobs:
     - name: Install pytorch 1.5.1 if python 3.6 (mac)
       if: ${{ matrix.python-version == '3.6' && matrix.os == 'macos-10.15' }}
       run: pip install torch==1.5.1
-    - name: Install pytorch 1.8.0 if python > 3.6 (mac)
+    - name: Install pytorch 1.10.0 if python > 3.6 (mac)
       if: ${{ matrix.python-version != '3.6' && matrix.os == 'macos-10.15' }}
-      run: pip install torch==1.8.0
-    - name: Install pytorch 1.5.1+cpu if python 3.6 (not mac)
```

Comment:
@interesaaat I left the line 42 to be torch 1.5.1 on old python on mac, and push all else forward.  Do you think we should leave others as well? (Ex: I can set line 47 back to 1.5 if we should test it)

Reply:
Yea I think we had that because 1.5.1 is only supported up to 3.6. In practice we should have a mix between what we had before and what you are proposing now. So 1.5.1 for 3.6, 1.10 for >= 3.6.